### PR TITLE
Migrate bicep example: 201/aci-wordpress

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Added bicep support to this sample by integrating bicep.main from the example in https://github.com/Azure/bicep/tree/main/docs/examples/201/aci-wordpress

The corresponding PR for the modified bicep example is here: https://github.com/Azure/bicep/pull/3310

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
> Status Message: Multiple error occurred: 'BadRequest':'InaccessibleImage':'The image 'wordpress:4.9-apache' in container group 'wordpress-containerinstance' is not accessible. Please check the image and registry credential.';'BadRequest':'InaccessibleImage':'The image 'mysql:5.6' in container group 'wordpress-containerinstance' is not accessible. Please check the image and registry credential.'. (Code:MultipleErrorsOccurred)

File: application-workloads/wordpress/aci-wordpress/main.bicep [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+resource miRoleAssign 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
+  name: roleAssignmentName
+  properties: {
+    roleDefinitionId: roleDefinitionId
+    principalId: mi.properties.principalId
+    principalType: 'ServicePrincipal'
+  }
+}
+
+var uamiId = resourceId(mi.type, mi.name)
```
Comment (bmoore-msft) [inline application-workloads/wordpress/aci-wordpress/main.bicep:47]: doesn't mi.id work here? or just remove it and use mi.id inline below?

File: application-workloads/wordpress/aci-wordpress/main.bicep [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+var scriptName = 'createFileShare'
+var identityName = 'scratch'
+
+resource mi 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+  name: identityName
+  location: location
+}
+
+var roleDefinitionId = resourceId('microsoft.authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
+var roleAssignmentName = guid(mi.name, roleDefinitionId, resourceGroup().id)
```
Comment (bmoore-msft) [inline application-workloads/wordpress/aci-wordpress/main.bicep:36]: move all the vars to a single "section" of the template

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Pull request merged**